### ISSUE TITLE
Cannot require other packages (phantomjs-node) installed globally

### ISSUE THREAD

**Issue by cloudgen** [notablemind/jupyter-nodejs#46]
I use the following script to install the kernel to jupyterhub , nodeJS version 4.2.6 , Jupyter 5.1 :
`
  cd ~
  git clone https://github.com/notablemind/jupyter-nodejs.git
  sudo mv ~/jupyter-nodejs /etc/
  cd /etc/jupyter-nodejs/
  sudo npm install 
  sudo npm run build
  sudo npm run build-ext
  cd /etc/jupyter-nodejs/
  sudo node install.js /usr/local/share/jupyter/kernels/nodejs/
`

Most of the functions works properly, except when I install phantomjs-node, I tried to require the this package in jupyterhub,

'
require('phantom');
' 
There is error message of file not found in context.js, and, if I modify the cell to 

'
require('/usr/local/lib/node_modules');
'
It works, what should I do if I want to require the package properly without specifying the full path of installed packages?

**Comment by jenningsanderson** [comment]
Yes, I just spent a long time debugging the same issue. Does jupyter-nodejs not by default look for global modules in `/usr/lib/node_modules` or `/usr/local/lib/node_modules`?

Further, I'm running it with jupyterhub, and it seems that whoever compiles it wins... meaning it was only looking in MY home directory's `node_modules` for modules; other users could not load modules from their local `node_modules` folder.

**Comment by Amertz08** [comment]
+1 Should maybe add directions for this to the docs imo.
